depthai_nodes/ml/messages/img_detections.py

```diff
@@ -40,44 +60,150 @@ def keypoints(
     @keypoints.setter
     def keypoints(
         self,
-        value: Keypoints,
-    ):
+        keypoints: List[Keypoint],
+    ) -> None:
         """Sets the keypoints.
 
         @param value: List of keypoints.
-        @type value: Keypoints
-        @raise TypeError: If the keypoints are not a Keypoints object.
+        @type value: List[Keypoint]
         """
-        if not isinstance(value, Keypoints):
-            raise TypeError("Keypoints must be a Keypoints object")
-        self._keypoints = value
+        self._keypoints = keypoints
```

Comment:
shouldn't we add some type checking if the value provided to `keypoints` is actually a list of `Keypoint` objects? Type checking is missing also in many other setters!